src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,38 @@
+package racingcar;
+
+public class Car {
+
+  private static final int DEFAULT_SCORE = 0;
+  private int score;
+
+  private final String name;
```

Comment:
```suggestion
  private final String name;
  private int score;
```

배치와 공백을 변경해보아도 좋을것같네요 😄